src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -527,3 +527,16 @@ def __update_token_patch(self, event, session):
         logging.debug(f"Event {event}, Session {session}")
         if session and event == "TOKEN_REFRESHED":
             self.client.postgrest.auth(session.access_token)
+
+    def set_cluster_active(self, active: bool) -> None:
+        try:
+            (
+                self.client.table(CLUSTERS_STATUS_TABLE)
+                .update({"active": active})
```

Comment:
I think it would be also good to update the `updated_at` column to `now()`, so it's easy to understand when it happened last